### PR TITLE
adds the "refresh TGUI" debug verb

### PR DESCRIPTION
**Pull request by azrael-x86**
## About The Pull Request

adds the `Refresh TGUI` verb to the ~~debug category~~ OOC category, ~~but it isn't locked behind any permissions. i'd put it in OOC, but.. eeeeeeh? there's not real much use for this verb other for debuggers.~~ i put it in the OOC category

## Why It's Good For The Game

debug purposes mostly

not player facing lole

### CODE REVIEW

**Comment by optimumtact** [comment]
`adds the Refresh TGUI verb to the debug category, but it isn't locked behind any permissions`
This will make an entire category appear for all users?

**Comment by magatsuchi** [comment]
> `adds the Refresh TGUI verb to the debug category, but it isn't locked behind any permissions`
> This will make an entire category appear for all users?

ah wait fook

**Review by stylemistake — COMMENTED** [review]
Line 373-375 in tgui_window.dm, remove sending of assets because they are already sent in reinitialize.

Update: I webedited it myself, double check.

File: code/modules/tgui/tgui_window.dm [hunk 1]
```diff
@@ -133,6 +133,9 @@
 		inline_js = initial_inline_js,
 		inline_css = initial_inline_css)
```
Comment (stylemistake) [inline code/modules/tgui/tgui_window.dm:135]:
```suggestion
```

**Review by stylemistake — APPROVED** [review]

**Comment by Iamgoofball** [comment]
cant you just press f5

File: code/modules/tgui_panel/external.dm [hunk 2]
```diff
@@ -35,3 +35,12 @@
 	// Force show the panel to see if there are any errors
 	winset(src, "output", "is-disabled=1&is-visible=0")
 	winset(src, "browseroutput", "is-disabled=0;is-visible=1")
+
+/client/verb/refresh_tgui()
+	set name = "Refresh TGUI"
+	set category = "OOC"
+
+	message_admins("start")
```
Comment (Kylerace) [inline code/modules/tgui_panel/external.dm:43]: epic

Comment (ZephyrTFA) [reply code/modules/tgui_panel/external.dm:43]: Local admins spammed with this one simple trick

**Pull request merged**